### ISSUE TITLE
About the operation of PMCNet

### ISSUE THREAD

**Issue by yaosx425** [GMuradas99/Symmetry-Detabase-Creator#2]
Hello! I saw that you started the GitHub repository for PMCNet. Did you run the code? Can it run?

**Comment by GMuradas99** [comment]
Hello!

I tried to run the code, I haven't been succesful yet.

Would you like to talk via e-mail?

**Comment by yaosx616** [comment]
> Hello!
> 
> I tried to run the code, I haven't been succesful yet.
> 
> Would you like to talk via e-mail?

Of course! My email is yaoshuxuan@stu.ouc.edu.cn . I first tried to test the pretrained weights provided by the author, but I found that he only output test loader and max F1, without reading the image for visualization of symmetry axis annotation. Then I tried to train my weights again, but I didn't know if it was a version issue with the downloaded package. I found that there were some undefined functions in his code, so I didn't know what to do.

**Comment by yaosx616** [comment]
> Hello!
> 
> I tried to run the code, I haven't been succesful yet.
> 
> Would you like to talk via e-mail?

I'm not sure if you've sent a message, but I'm afraid there might be a problem with my email and I won't receive any messages（*/ ω＼*)

**Comment by GMuradas99** [comment]
I just sent you an email,
did you recieve it?
